test/test_api/test_api.py

```diff
@@ -27,8 +27,8 @@
 # Test
 # ========
 @pytest.mark.parametrize('openml_id', (40981, ))
-@pytest.mark.parametrize('resampling_strategy', (HoldoutValTypes.holdout_validation,
-                                                 CrossValTypes.k_fold_cross_validation, ))
+@pytest.mark.parametrize('resampling_strategy', (CrossValTypes.k_fold_cross_validation,
```

Comment:
why removing holdout?

Reply:
:P I'll fix this, I did this for debugging